runtime/kusama/src/lib.rs

```diff
@@ -558,13 +558,33 @@ impl pallet_staking::Config for Runtime {
 	type WeightInfo = weights::pallet_staking::WeightInfo<Runtime>;
 }
 
+macro_rules! prod_or_test {
+	($prod:expr, $test:expr) => {
+		if cfg!(feature = "fast-runtime") {
+			$test
+		} else {
+			$prod
+		}
+	};
+	($prod:expr, $test:expr, $env:expr) => {
+		if cfg!(feature = "fast-runtime") {
+			core::option_env!($env)
+				.map(|s| s.parse().ok())
+				.flatten()
+				.unwrap_or($test)
+		} else {
+			$prod
+		}
+	}
+}
+
 parameter_types! {
-	pub const LaunchPeriod: BlockNumber = 7 * DAYS;
-	pub const VotingPeriod: BlockNumber = 7 * DAYS;
-	pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
+	pub LaunchPeriod: BlockNumber = prod_or_test!(7 * DAYS, 1, "KSM_LAUNCH_PERIOD");
+	pub VotingPeriod: BlockNumber = prod_or_test!(7 * DAYS, 1 * MINUTES, "KSM_VOTING_PERIOD");
+	pub FastTrackVotingPeriod: BlockNumber = prod_or_test!(3 * HOURS, 1 * MINUTES, "KSM_FAST_TRACK_VOTING_PERIOD");
```

Comment:
What do you think of this?
@kianenigma @shawntabrizi 
Allows specifying parameter types via env variable in case the `fast-runtime` feature is activated.

Reply:
looks good to me.

all in all, I think this is a good change. I deal with this personally as well at times and always have to manually tweak the parameters. That being said, I think we shouldn't over do it and only apply this feature to a small number of obvious ones. The reason would be that it is hard to choose a good number for "what is fast" for every parameter.

Reply:
That's why I figured the env variables would be good, because they allow for easy tweaking at build time.